tests/linux-sandbox.sh

```diff
@@ -34,19 +34,23 @@ nix-build dependencies.nix --no-out-link --check --sandbox-paths /nix/store
 # Test that sandboxed builds with --check and -K can move .check directory to store
 nix-build check.nix -A nondeterministic --sandbox-paths /nix/store --no-out-link
 
-(! nix-build check.nix -A nondeterministic --sandbox-paths /nix/store --no-out-link --check -K 2> $TEST_ROOT/log)
-if grepQuiet 'error: renaming' $TEST_ROOT/log; then false; fi
-grepQuiet 'may not be deterministic' $TEST_ROOT/log
+expectStderr 104 nix-build check.nix -A nondeterministic --sandbox-paths /nix/store --no-out-link --check -K \
+  | tee >( grepQuietInverse 'error: renaming' ) \
```

Comment:
Not quite exactly, totally, completely nor absolutely sure that this preserves the old semantics.

Reply:
Sadly, I know no way to test the tests :-)